pages/_next-prefetcher.js

```diff
@@ -0,0 +1,108 @@
+/* global self */
```

Comment:
Can you move this file to `client` dir ?
`pages` directory is intended to be for default pages like `/_error`.